src/top/tensor/elemwise.cc

```diff
@@ -247,6 +248,25 @@ NNVM_REGISTER_INIT_OP(fill)
 )code"  NNVM_ADD_FILELINE)
 .set_support_level(1);
 
+// Generate constant zero
+NNVM_REGISTER_INIT_OP(__zero__)
+.describe(R"code(Return constant value zero
+
+)code"  NNVM_ADD_FILELINE)
+.set_support_level(1);
+
+NNVM_REGISTER_INIT_OP(_zeros)
```

Comment:
what's the difference between __zero__ and _zeros?

Reply:
I think Yao is using __zero__ to construct a constant zero scalar.
I think we can use _zeros to do the same.

Reply:
one thing we need to make sure is to be consistent as numpy.